Documentation/grpc-metadata.md

```diff
@@ -223,3 +223,60 @@ func (s *server) SomeStreamingRPC(stream pb.Service_SomeStreamingRPCServer) erro
     stream.SetTrailer(trailer)
 }
 ```
+
+## Updating metadata from a server interceptor
+
+An example for updating metadata from a server interceptor is
+available [here](../examples/features/metadata_interceptor/server/main.go).
+
+#### Unary interceptor
+
+The interceptor can read the existing metadata from the RPC context passed to it.
```

Comment:
Sorry, maybe I suggested this verbiage earlier. But maybe, we can skip one of the `the`s. So, it will be:

```
The interceptor can read existing metadata from the RPC context passed to it.
```

Reply:
Done